R/data.R

```diff
@@ -15,3 +15,15 @@
 #'
 #' @format A tibble with a row per issue date
 "jhu_cases_data"
+
+
+#' imputed deaths
+#'
+#' @format A data.frame
+"jhu_deaths_imputed_data"
+
+#' imputed cases
+#'
+#' @format A data.frame
```

Comment:
Let's add a little more detail to this description: what are the variables in the data frame?